PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Application/Services/User/UserService.cs

```diff
@@ -0,0 +1,80 @@
+using AutoMapper;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Identity;
+using PrivateHospitals.Application.Dtos.Doctor;
+using PrivateHospitals.Application.Dtos.Patient;
+using PrivateHospitals.Application.Dtos.User;
+using PrivateHospitals.Application.Interfaces;
+using PrivateHospitals.Application.Interfaces.User;
+using PrivateHospitals.Application.Responses;
+using PrivateHospitals.Core.Models;
+using PrivateHospitals.Data.Interfaces.User;
+
+namespace PrivateHospitals.Application.Services.User;
+
+public class UserService(
+    IUserRepository userRepository,
+    SignInManager<AppUser> signInManager,
+    ITokenService tokenService,
+    IMapper mapper
+): IUserService
+{
+    private readonly IUserRepository _userRepository = userRepository;
+    private readonly SignInManager<AppUser> _signInManager = signInManager;
+    private readonly ITokenService _tokenService = tokenService; 
+    private readonly IMapper _mapper = mapper;
```

Comment:
They are already declared in your primary constructor

Reply:
Fixed